integrations/terraform/test/role_test.go

```diff
@@ -252,14 +252,14 @@ func (s *TerraformSuite) TestRoleLoginsSplitBrain() {
 }
 
 func (s *TerraformSuite) TestRoleVersionUpgrade() {
+	s.T().Skip("Test temporarily disabled until v16")
```

Comment:
Maybe leave a TODO here to help remind us?

```suggestion
	// TODO(hugoShaka): Revisit TestRoleVersionUpgrade after v16.
	s.T().Skip("Test temporarily disabled until v16")
```